packages/react-mutation-mapper/src/component/track/TrackItem.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { computed, makeObservable, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import TrackCircle from './TrackCircle';
+import TrackRect from './TrackRect';
+import ExonNumTrack from './ExonNumTrack';
+
+type TrackItemProps = {
+    x: number;
+    y: number;
+    dim1?: number; // radius or width
+    dim2?: number; // hoverRadius or height
```

Comment:
This is a bit confusing. We should come up with something better. Ideally we should also allow hover dimensions for rectangles. Current implementation only allows to define hover radius for circles, but no hover width or height for rectangles.